inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -986,14 +986,29 @@ void InsertSplitAligningFilterPass::run() {
                 CNNNetworkInsertLayer(l, nullptr, filterWithQuant, splitOutIndex);
             }
 
-
             // search data that starts from unaligned location
             currentOffset += outputSize * bytesPerSplitElement;
             splitOutIndex++;
         }
     }
 }
 
+static InferenceEngine::Blob::Ptr tileBlob(Blob::Ptr& blob, size_t TileTo) {
```

Comment:
having function that cannot work without external  gdc checking makes it less universal, also static functions cannot be tested under unit tests environment. I recommend having proper self contain implementation of tile utility function.